pkg/adaptor/elasticsearch/clients/v2/writer.go

```diff
@@ -0,0 +1,91 @@
+package v2
```

Comment:
I know you're aware of all the code duplication, just want to make sure we track it in an issue to investigate clever ways to do the versioning.